AccessibilitySnapshot.podspec

```
@@ -15,33 +15,30 @@ Pod::Spec.new do |s|
   s.default_subspecs = 'Core', 'SnapshotTesting'
 
   s.subspec 'Core' do |ss|
-    ss.source_files = 'AccessibilitySnapshot/Core/Classes/**/*.{swift,h,m}'
-    ss.public_header_files = [
-      'AccessibilitySnapshot/Core/Classes/UIAccessibilityStatusUtility.h',
-      'AccessibilitySnapshot/Core/Classes/UIView+DynamicTypeSnapshotting.h',
-    ]
+    ss.source_files = 'Sources/AccessibilitySnapshot/Core/Swift/Classes/**/*.swift', 'Sources/AccessibilitySnapshot/Core/ObjC/**/*.{h,m}'
+    ss.public_header_files = 'Sources/AccessibilitySnapshot/Core/ObjC/include/*.h'
     ss.resource_bundles = {
-     'AccessibilitySnapshot' => ['AccessibilitySnapshot/Core/Assets/**/*.{strings,xcassets}']
+     'AccessibilitySnapshot' => ['Sources/AccessibilitySnapshot/Core/Swift/Assets/**/*.{strings,xcassets}']
     }
   end
 
   s.subspec 'iOSSnapshotTestCase' do |ss|
-    ss.source_files = 'AccessibilitySnapshot/iOSSnapshotTestCase/Classes/**/*.{swift,h,m}'
+    ss.source_files = 'Sources/AccessibilitySnapshot/iOSSnapshotTestCase/**/*.{swift,h,m}'
     ss.public_header_files = [
-      'AccessibilitySnapshot/iOSSnapshotTestCase/Classes/FBSnapshotTestCase_Accessibility.h',
+      'Sources/AccessibilitySnapshot/iOSSnapshotTestCase/FBSnapshotTestCase_Accessibility.h',
     ]
 
     ss.dependency 'AccessibilitySnapshot/Core'
     ss.dependency 'iOSSnapshotTestCase', '~> 6.0'
   end
 
   s.subspec 'SnapshotTesting' do |ss|
-    ss.source_files = 'AccessibilitySnapshot/SnapshotTesting/Classes/**/*.{swift,h,m}'
+    ss.source_files = 'Sources/AccessibilitySnapshot/SnapshotTesting/**/*.{swift,h,m}'
 
     ss.dependency 'AccessibilitySnapshot/Core'
     ss.dependency 'SnapshotTesting', '~> 1.0'
   end
 
   s.frameworks = 'XCTest'
   s.weak_frameworks = 'XCTest'
-end
+end
```

nit: Leave the newline at the end of the file
```suggestion
end

```